src/components/map/ComponentLink.js

```diff
@@ -46,8 +46,8 @@ function ComponentLink(props) {
 					link={link}
 					metaText={props.metaText}
 					setMetaText={props.setMetaText}
-					x={x2}
-					y={y2}
+					x={x2.toString()}
+					y={y2.toString()}
```

Comment:
FlowText only uses these as Strings internally. I think this could probably go either way, either do what I have done, or change the PropTypes to only want Numbers, and convert internally.